keps/sig-apps/2185-random-pod-select-on-replicaset-downscale/README.md

```diff
@@ -243,53 +243,63 @@ _This section must be completed when targeting alpha to a release._
     - Will enabling / disabling the feature require downtime or reprovisioning
       of a node?
 
-* **Does enabling the feature change any default behavior?**
+###### Does enabling the feature change any default behavior?
   Yes, this changes the default assumption that the youngest pod in a replica set 
   will always be the one evicted. However, it still groups pods by their age and picks 
   from the youngest group.
 
-* **Can the feature be disabled once it has been enabled (i.e. can we roll back
-  the enablement)?**
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
   Yes. Existing workloads should see no change when disabling this feature.
 
-* **What happens if we reenable the feature if it was previously rolled back?**
+###### What happens if we reenable the feature if it was previously rolled back?
   Assumptions that the newest pod will be deleted first may break.
 
-* **Are there any tests for feature enablement/disablement?**
+###### Are there any tests for feature enablement/disablement?
   Tests for feature disablement shouldn't be necessary, as this is already an assumed 
   (but not documented) controller behavior.
 
 ### Rollout, Upgrade and Rollback Planning
 
 _This section must be completed when targeting beta graduation to a release._
 
-* **How can a rollout fail? Can it impact already running workloads?**
+###### How can a rollout or rollback fail? Can it impact already running workloads?
   This should not affect running workloads, though there is the possibility that the logic 
   panics which would cause kube-controller-manager to crash
 
-* **What specific metrics should inform a rollback?**
+###### What specific metrics should inform a rollback?
   Increased pod deletions could indicate runaway/hot-loop failures in the scaledown logic.
   Availability of applications may also be affected. Though the intent of this is to provide 
   better available through more distributed victim selection, in cases of desired binpacking 
   pods may remain running on undesired nodes.
 
-* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
   This will be manually tested before the graduation to beta
```

Comment:
Uhm... I can't remember if we did this manual testing.
@damemi do you have any insights?

But is it still worth doing the manual testing if it won't be possible to opt-out from the feature?

Reply:
I have revised the other two suggestions. I will wait @damemi's response for this.

Reply:
This is purely in-memory change for the controller, so upgrade/downgrade doesn't really change anything.

So let's just switch to sth like that.